openlibrary/core/vendors.py

```diff
@@ -1,11 +1,8 @@
 import re
-import web
-import urllib2
 import simplejson
 import requests
 from decimal import Decimal
 from amazon.api import SearchException
-from infogami import config
```

Comment:
@hornc @mekarpeles Do either of you know if this has side-effects? I can't recall which one of our configs has side-effects, but removing it might cause things to break in unexpected ways. It was added as part of this commit: https://github.com/internetarchive/openlibrary/commit/bdcfe0e24feae560f5987c32ce1c10925068ca5d , which also doesn't use `config` directly, so my guess would be it has side-effects related to `web` and `accounts` access.

Reply:
I don't believe it has side effects. I certainly hope it doesn't.

I thought it was added with the assumption that this code would need to make requests to archive.org, but now requests to archive.org are correctly handled by the `ia.py` module. I know I have refactored some other code recently and made sure the requests were done by `ia.py` which removed the need for getting the config value from elsewhere.

Heh, the other reason why this isn't used when it could have been that other config is being pulled from `lending.config_ia_ol_metadata_write_s3` ... the lending module's config hack :allthethings:  (which is not my favourite)

Reply:
Tested this on dev ( https://dev.openlibrary.org/admin/status ); working ✅